README.md

```diff
@@ -115,7 +115,7 @@ along with this software. If not, see [CC0][].
 * `ed25519/ed25519-donna` adopted from [ed25519-donna][]
 * Idea used in `worker_fast()` is stolen from [horse25519][]
 * base64 routines and initial YAML processing work contributed by
-  Alexander Khristoforov (heios@protonmail.com)
+  Alexander Khristoforov (heios at protonmail dot com)
```

Comment:
That doesn't matter and only annoys people. Email scrapper are smart enough to catch this.

Reply:
> That doesn't matter

then it's not worse.

> only annoys people

like I said before this is more of credit, than invitation to send emails. annoying anyone who wants to send using this is a feature.

> Email scrapper are smart enough to catch this.

perhaps. I wouldn't be surprised if some, maybe even the most of them could parse that.
but there also going to be some which won't parse this.
However, `<a href="mailto:...` github generates is probably easiest to parse compared to anything else.

I would sorta prefer anything than letting github linkifying it.
Because its mailto doesn't include realname.
Because it helps scrapers.
Because it's not intended to be invitation of sending mail.
Because it's just ugly design to take any text looking like email address and making it into actual link without proper markup explicitly requesting such behavior.

Reply:
> then it's not worse.

It is worse for humans because you can just copy the mail.

>  Because it helps scrapers.

Let me test something:

```suggestion
  Alexander Khristoforov (heios@protonmail.com)
```

> Because it's just ugly design to take any text looking like email address and making it into actual link without proper markup explicitly requesting such behavior.

Convenience is not always ugly. Sometimes it catches wrong stuff but then you just put it into code blocks.

Reply:
I think we should leave it like it is now. If we change it back, then we should also change the address of cathugger (in the text above the onion link). If people want to write a mail, they can go trough the effort of changing the format or visit the onion link or use gpg.